tests/test_benchmark.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+from os import path
+from conftest import skipif
+from devito import configuration
+from subprocess import check_call
+
+
+pytestmark = skipif(['yask', 'ops'])
+
+
+@pytest.mark.parametrize('mode', ['bench'])
+@pytest.mark.parametrize('problem', ['acoustic', 'tti', 'elastic', 'viscoelastic'])
+def test_bench(mode, problem):
+    """
+    Test combinations of modes and problems.
+    Testing for both OpenMP enabled or disabled.
+    """
+    if configuration['openmp']:
+        nthreads = configuration['platform'].cores_physical
+    else:
+        nthreads = 1
+
+    command_bench = ['python', 'benchmarks/user/benchmark.py', mode, '-P', problem,
+                     '-d', '16', '16', '16', '--tn', '4', '-x', '1']
+    check_call(command_bench)
```

Comment:
did you verify that if you for example supply an unknown flag to the command , the test actually fails ?

For example:
```
['python', 'benchmarks/user/benchmark.py', mode, '-P', problem, ......, '-asdjoias', ...]
```

and, analogously, if there were an error in the execution of the benchmark, the test fails as well?

just to be sure...

Reply:
Yes, it does fail, I have tested mispellings, wrong args/unkonwn flags, Well this how we realized the error that you fixed in https://github.com/devitocodes/devito/pull/1080

Reply:
good